src/main/resources/db/changelog/changesets/changelog_20230817T145909Z.xml

```diff
@@ -0,0 +1,94 @@
+                <constraints nullable="false" primaryKey="true" primaryKeyName="volatge_limits_parameterPK"/>
+            </column>
+            <column name="high_voltage_limit" type="FLOAT8"/>
+            <column name="low_voltage_limit" type="FLOAT8"/>
+            <column name="priority" type="INT"/>
+            <column name="voltage_init_setting_id" type="UUID"/>
+        </createTable>
+    </changeSet>
+    <changeSet author="labidiayo (generated)" id="1692284360171-2">
+        <createTable tableName="voltage_init_setting">
+            <column name="id" type="UUID">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="voltage_init_settingPK"/>
+            </column>
+            <column name="date" type="TIMESTAMP"/>
+            <column name="name" type="VARCHAR(255)"/>
+        </createTable>
+    </changeSet>
+    <changeSet author="labidiayo (generated)" id="1692284360171-3">
+        <createTable tableName="voltage_init_setting_entity_constantqgenerators">
+            <column name="voltage_init_setting_id" type="UUID">
+                <constraints nullable="false"/>
+            </column>
+            <column name="filter_id" type="UUID"/>
+            <column name="filter_name" type="VARCHAR(255)"/>
+        </createTable>
+    </changeSet>
+    <changeSet author="labidiayo (generated)" id="1692284360171-4">
+        <createTable tableName="voltage_init_setting_entity_variable_shunt_compensators">
+            <column name="voltage_init_setting_id" type="UUID">
+                <constraints nullable="false"/>
+            </column>
+            <column name="filter_id" type="UUID"/>
+            <column name="filter_name" type="VARCHAR(255)"/>
+        </createTable>
+    </changeSet>
+    <changeSet author="labidiayo (generated)" id="1692284360171-5">
+        <createTable tableName="voltage_init_setting_entity_variable_two_wt">
+            <column name="voltage_init_setting_id" type="UUID">
+                <constraints nullable="false"/>
+            </column>
+            <column name="filter_id" type="UUID"/>
+            <column name="filter_name" type="VARCHAR(255)"/>
+        </createTable>
+    </changeSet>
+    <changeSet author="labidiayo (generated)" id="1692284360171-6">
+        <createTable tableName="voltage_limits_parameter_entity_filters">
+            <column name="voltage_limit_id" type="UUID">
+                <constraints nullable="false"/>
+            </column>
+            <column name="filter_id" type="UUID"/>
+            <column name="filter_name" type="VARCHAR(255)"/>
+        </createTable>
+    </changeSet>
+    <changeSet author="labidiayo (generated)" id="1692284360171-7">
+        <addForeignKeyConstraint baseColumnNames="voltage_init_setting_id" baseTableName="volatge_limits_parameter" constraintName="FK4vm0s14orh2rdrar4nubva2bj" deferrable="false" initiallyDeferred="false" referencedColumnNames="id" referencedTableName="voltage_init_setting" validate="true"/>
```

Comment:
I reckon there is a missing index for this foreign key for this column
Additionally the foreign key could benefit from a more explicit name